### ISSUE TITLE
无法识别pics目录里面带猫的图片

### ISSUE THREAD

**Issue by sunjl** [wikke/SSD_Keras#1]
下载VOC2007和PASCAL_VOC_SSD300_weight_checkpoint-epoch-16-loss-3.3392.hdf5文件，运行SSD.py，可以识别自行车、人物，图片上都能正确标注区域，但是所有带猫的图片没有标注。

我自己fork了你的项目，[https://github.com/sunjl/keras_ssd](https://github.com/sunjl/keras_ssd)，修改了SSD.py文件：
```
OUTPUT_DIR = './output'
MODEL_H5_NAME = 'weight.hdf5'
# MODEL_H5_NAME = 'PASCAL_VOC_SSD300_weight_checkpoint-epoch-16-loss-3.3392.hdf5'
model.save_weights(os.path.join(OUTPUT_DIR, MODEL_H5_NAME))
model.load_weights(os.path.join(OUTPUT_DIR, MODEL_H5_NAME))
```
自己训练模型，无法识别pics任一图片，怎么回事？
运行环境：Ubuntu 16.04，CUDA 8.0，Python 3.5，Keras 2.0.9，Tensorflow 1.3.0
GTX960显卡，4G显存不够用，所以把SSD.py文件中的batch改为1

**Comment by wikke** [comment]
1、我的ipynb里也没有把猫框出来。也许设置阈值就可以出来吧，不过可能候选就很多了，也失去意义。
2、我是阿里云比较高的配置训练了几个小时吧。batch是1本身随机性就很大，需要更小的learning rate和更多的epochs。建议你升级配置，训练更长的时间。
